client/src/components/generic/SignUpNotif/SignUpNotif.tsx

```diff
@@ -22,15 +23,20 @@ export const SignUpNotif = ({ signedIn }: ISignUpNotif) => {
 
   return (
     <div
-      className="text-dark-blue-100 border-gray-3 flex flex-col items-center justify-center 
-    gap-4 rounded-md border bg-white py-8"
+      className="border-gray-3 flex max-h-[402px] max-w-[570px] flex-col items-center justify-center 
```

Comment:
the problem with constraining in here means that it is not reusable, generally the way to do this is allow containers to stretch max width/height, and use a parent container to constrain it

Reply:
I see that fixed the issue; thanks for the tip.